### ISSUE TITLE
Documentation update

### ISSUE THREAD

**Issue by evromalarkey** [phonegap/phonegap-plugin-push#2081]
Does this plugin now require FCM? It's not very clear to me from the installation doc.
I can see that there is now requirement to have google-services.json, but that is only possible with FCM right?
You should consider adding into the docs how to even start with FCM (or that it's required), because as a newcomer from version 1.x I have no idea where to start.

Also is this documentation still actual or was that only for 1.x version? If so, it should be also updated - http://docs.phonegap.com/tutorials/develop/push-notifications/

**Comment by macdonst** [comment]
@evromalarkey yes, the docs at http://docs.phonegap.com/tutorials/develop/push-notifications/ are up date. You won't need the google-services.json file for that tutorial as it is already included in the PhoneGap Developer App.

Yes, 2.x uses FCM and 1.x uses GCM.

**Comment by evromalarkey** [comment]
Yet in the installation doc it is not clear to me how to even start with GCM (or that I have to). Also that tutorial on the phonegap.com should have info that it's only for version 1.x, because in foot it redirects the user to documentation for 2.x

I understand it now, but as a newcomer from that tutorial, the documentation was little confusing for me,

**Comment by macdonst** [comment]
@evromalarkey yeah, I don't actually want folks to use GCM as it is deprecated by Google. FCM is the way to go. 

I'm always looking to improve the docs. How can I re-word the warning so that folks know that 2.x is for FCM and 1.x is for GCM?

That tutorial works fine with the 2.x version of the plugin as that is what is included in the dev app and our push service sends via FCM.

**Comment by evromalarkey** [comment]
Yeah FCM is definitely better, the thing is that in that tutorial, it downloads template with version 1.8. And there is the part about sender_id (which is GCM), but that is replaced with google-services.json in 2.x (FCM). Maybe point out the differences between 1.x and 2.x versions in that tutorial?

Also in the installation documentation it would be good to point out how to get google-services.json, that you need to setup it in Firebase console. I think that this was more clear in 1.x documentation with the sender_id.

To be honest I didn't know much about FCM, so I needed to googling a bit before I learned that I need to setup google-services.json in Firebase console etc.
